purldb-toolkit/src/purldb_toolkit/purlcli.py

```diff
@@ -27,47 +29,129 @@ def purlcli():
     type=click.File(mode="w", encoding="utf-8"),
     required=True,
     default="-",
-    help="Write validation output as JSON to FILE.",
+    help="Write versions output as JSON to FILE.",
 )
 @click.option(
     "--file",
     type=click.File(mode="r", encoding="utf-8"),
     required=False,
     help="Read a list of PURLs from a FILE, one per line.",
 )
-def validate(purls, output, file):
+def get_meta(purls, output, file):
     """
-    Check the syntax of one or more PURLs.
+    Given one or more PURLs, for each PURL, return a mapping of metadata
+    fetched from the fetchcode package.py info() function.
     """
-    if purls and file:
-        raise click.UsageError("Use either purls or file but not both.")
-
-    if not (purls or file):
-        raise click.UsageError("Use either purls or file.")
+    check_for_duplicate_input_sources(purls, file)
 
     if file:
         purls = file.read().splitlines(False)
 
-    validated_purls = validate_purls(purls)
-    json.dump(validated_purls, output, indent=4)
+    context = click.get_current_context()
+    manual_command_name = context.command.name
```

Comment:
I think `command_name` would be alright.

Reply:
Done.